corehq/ex-submodules/casexml/apps/case/templatetags/case_tags.py

```diff
@@ -226,6 +229,20 @@ def process_output(case_output, depth=0):
 
 
 def get_case_hierarchy(case, type_info):
+
+    def get_all_reverse_indices(case):
+        """
+        Return a list of child and extension type reverse indices for the given
+        case
+        """
+        children_indices = case.reverse_indices
```

Comment:
Yeah, I don't think there should be any issues if you made `get_reverse_indices` return extension cases too, actually. I don't think I had a reason to default to `CHILD_RELATIONSHIP` other than just wanting to retain functionality the same, since extension cases was only potentially going to be used for the ICDS app. 
In theory it should just work if you follow the same pattern as in `get_all_reverse_indices_info`